packaging/debian-sid/snapd.postrm

```diff
@@ -96,6 +96,10 @@ if [ "$1" = "purge" ]; then
         rm -f "/etc/systemd/system/multi-user.target.wants/$unit"
     done
 
+    # now stop snapd itself to ensure we can clearly remove all snapd data
+    systemctl_stop snapd.socket || true
+    systemctl_stop snapd.service || true
```

Comment:
Hmm, something does not add up. This is postrm, so if I understand how things work in debs, snapd.service and snapd.socket and all service files provided by the package should be long gone by now? The snapd service/socket, snap failure all daemon should have been stopped/masked/disabled in prerm and then removed. Perhaps we are missing a different step in prerm?